src/current/_includes/molt/replicator-flags.md

```diff
@@ -2,44 +2,44 @@
+| `--stagingConn`              | `STRING`   | The staging database's connection string.                                                                                                                                                                                                                                                                                                             |
+| `--stagingCreateSchema`      |            | Automatically create the staging schema if it does not exist.                                                                                                                                                                                                                                                                                         |
+| `--stagingIdleTime`          | `DURATION` | Maximum lifetime of an idle connection.<br><br>**Default:** `1m0s`                                                                                                                                                                                                                                                                                    |
+| `--stagingJitterTime`        | `DURATION` | The time over which to jitter database pool disconnections.<br><br>**Default:** `15s`                                                                                                                                                                                                                                                                 |
+| `--stagingMaxLifetime`       | `DURATION` | The maximum lifetime of a database connection.<br><br>**Default:** `5m0s`                                                                                                                                                                                                                                                                             |
+| `--stagingMaxPoolSize`       | `INT`      | The maximum number of staging database connections.<br><br>**Default:** `128`                                                                                                                                                                                                                                                                         |
+| `--stagingSchema`            | `ATOM`     | Name of the SQL database schema that stores replication metadata. **Required** each time [`--mode replication-only`](#replicate-changes) is rerun after being interrupted, as the schema provides a replication marker for streaming changes. For details, refer to [Replicate changes](#replicate-changes).<br><br>**Default:** `_replicator.public` |
```

Comment:
Isn't the `--stagingSchema` basically a STRING of a certain format? Why is it represented as "ATOM"? That's quite confusing.

Reply:
The types are 1:1 from the original Replicator doc, as I didn't have time to verify whether we could use alternatives. But I'll definitely amend this one if you're confirming that it's a `STRING`.

Reply:
I'll amend `--targetSchema` to a `STRING` type as well, while I'm here.

Reply:
Thanks, this will be a lot clearer now. The example also shows the string format, which is helpful.